src/cr/cube/matrix/assembler.py

```diff
@@ -194,11 +194,16 @@ def pairwise_means_indices(self, alpha, only_larger):
 
         return np.array(
             [
-                tuple(col_ord_map[idx] for idx in idxs) if idxs is not None else None
-                for idxs in assembled_matrix.ravel()
+                [
+                    tuple(col_ord_map[idx] for idx in idxs)
```

Comment:
If we changed the name of `col_ord_map` to just `ord_map`, so that it's a bit shorter, the display of the list comprehension would be much more pretty.